gnat2goto/COMPILING.md

```diff
@@ -47,3 +47,44 @@ Assuming that you've built the dependencies as per the above instructions
 This should build into install/bin. You can test if it works by calling:
 
 ` $ install/bin/gnat2goto examples/foo.adb`
+
+# Building and running gnat2goto unit tests
+
+gnat2goto includes a selection of unit tests. These are built as a seperate
+`unit_tests` binary. Assuming you have built gnat2goto as per the above
+instructions, the `unit_tests` binary can be built with the following command
+
+`$ gprbuild -P unit_tests.gpr -cargs -largs`
```

Comment:
What is this `-cargs` and `-largs` stuff? I've never used those when building the unit tests (also worth pointing out that you don't need to rebuild gnat2goto all the time, you only need to invoke the `Makefile` again if you changed the irep specs)

Reply:
For the -cargs and -largs - don't honestly know, but thats what is being run in CI.